evm_loader/lib/src/solana_simulator/utils.rs

```diff
@@ -21,8 +21,15 @@ use solana_sdk::{
 
 use crate::rpc::Rpc;
 
+#[derive(Eq, PartialEq, Copy, Clone)]
+pub enum SyncState {
```

Comment:
Maybe `SyncSysvars` could be better? `State` can be too general.

Reply:
`SyncSysvars` was my first choice for this. 
But it's not just sysvars. It affects features as well.

Reply:
Yes, sorry, I missed the usage for features